pkg/detectors/ftp/ftp_test.go

```diff
@@ -48,7 +51,7 @@ func TestFTP_FromChunk(t *testing.T) {
 				{
 					DetectorType: detectorspb.DetectorType_FTP,
 					Verified:     true,
-					Redacted:     "ftp://dlpuser:*************************@ftp.dlptest.com",
+					Redacted:     "ftp://dlpuser:********@ftp.dlptest.com",
```

Comment:
Curious why some of the tests were adjusted? Did our redaction code update?

Reply:
i guess so? they were failing on `main` and i didn't think that figuring out why was worth it